packages/sources/finage-test/src/endpoint/forex.ts

```diff
@@ -4,6 +4,35 @@ import overrides from '../config/overrides.json'
 import { httpTransport } from '../transport/forex-http'
 import { ForexBaseEndpointTypes, forexPriceInputParameters } from './utils'
 import { wsTransport } from '../transport/forex-ws'
+import { AdapterRequest } from '@chainlink/external-adapter-framework/util'
+import { PriceAdapterRequest } from '../index'
+
+/* 
+excludesMap includes currency pairs {quote: [base ...]} which should not be inverted.
+*/
+const excludesMap: Record<string, string[]> = {
+  USD: ['EUR', 'GBP', 'AUD', 'NZD', 'XAG', 'XAU', 'XPT', 'XPD', 'XCU'],
+}
+
+export const forexReqTransformer = (req: AdapterRequest<any>): void => {
+  if (req.requestContext.endpointName == 'forex') {
+    const priceRequest = req as PriceAdapterRequest<any>
+
+    const quote = req.requestContext.data.quote
+    const base = req.requestContext.data.base
+    let inverse
+    if (excludesMap?.[quote].includes(base)) inverse = false
```

Comment:
This should also convert the `quote` and `base` to uppercase so that the match the `excludesMap` - I think this might be why the integ tests are failing even when adding fixtures


```suggestion
    if (excludesMap[quote.toUpperCase()].includes(base.toUpperCase())) {
        inverse = false
    }
```